bridge-adaptivity/api/backends/openedx.py

```diff
@@ -60,8 +62,11 @@ def get_oauth_access_token(self):
             log.exception(
                 'OAuth2 token request to the OpenEdx LTI Provider failed: {}'.format(exc.message)
             )
+            log.warn("You may want to check your OAuth registration on LTI Provider."
```

Comment:
Why do we need two logging messages if the exception is raised one well-informed message would be fair enough.